misc/images/materialized/entrypoint.sh

```diff
@@ -28,6 +28,7 @@ hosted offering we run these services scaled across many machines.
 ********************************* WARNING ********************************
 EOF
 
+if [ -z "${MZ_NO_BUILTIN_COCKROACH:-}" ]; then
```

Comment:
@benesch can you confirm that you are OK with this (given that this affects the stock materialized container we distribute)

Reply:
Not entirely sure why you need it (builtin Cockroach is ignored if you pass in different persist/stash URLs, no?) but seems fine!

Reply:
It is ignored by Mz, but it is still stated, so it takes exclusive locks in its data directory.